agent_battler/agents.py

```
@@ -0,0 +1,236 @@
+
+    async def execute(self, instruction: str, proxy_env: Dict[str, str]) -> int:
+        """Execute auggie CLI with the instruction."""
+        env = os.environ.copy()
+        env.update(proxy_env)
+
+        # Node.js v12+ supports NODE_USE_ENV_PROXY=1 to enable built-in proxy support
+        # This makes Node.js respect HTTP_PROXY and HTTPS_PROXY environment variables
+        env['NODE_USE_ENV_PROXY'] = '1'
+        env['NODE_TLS_REJECT_UNAUTHORIZED'] = '0'  # Disable SSL verification for proxy
+
+        print(f"🔧 Configuring Node.js to use proxy (NODE_USE_ENV_PROXY=1)")
```

Ruff (0.14.3)

f-string without any placeholders

Remove extraneous `f` prefix

(F541)